Exercise2-Group Hinata.R

```diff
@@ -0,0 +1,24 @@
+# Install and load necessary packages
+install.packages("foreign")
+library(foreign)
+install.packages("gtsummary")
+library(gtsummary)
+library(ggplot2)
+
+# Read the fem.dat dataset
+fem_data <- read.table("https://raw.githubusercontent.com/OxfordIHTM/teaching_datasets/main/fem.dat", header = TRUE)
+
+# Summary of variables for AGE, IQ, and WT
+summary(fem_data$AGE)
+summary(fem_data$IQ)
+summary(fem_data$WT)
+
+# Summary in tabular form using gtsummary
+summary_table <- tbl_summary(fem_data)
+summary_table
+
+# Create boxplots for AGE, IQ, and WT
+par(mfrow=c(1,3))
+boxplot(fem_data$AGE, main="Age", col="skyblue", border="black")
+boxplot(fem_data$IQ, main="IQ", col="lightgreen", border="black")
+boxplot(fem_data$WT, main="Weight", col="lightcoral", border="black")
```

Comment:
Good. You are using `par()` to set graphical parameters to put the boxplots side by side. Because the ranges of values for these variables are different, it wouldn't really work to put them all in one graphical window using `boxplot()` function. Well done.

My main comment here is that everytime you change the graphical parameters using `par()` you should change it back to its original state. So, in the line after line 25, you should put this:

```R
par(mfrow = c(0,0))
```

Reply:
Dear @ernestguevarra 
I have made the changes however the last comment about par(mfrow = c(0,0)) is not working. Could you please help me identify the issue

Reply:
For as long as you wrote it correctly, which you did, you don't expect to see anything different as the change happens invisible (the change is at the environment level).

I checked your changes and it all looks good to me.